imap/annotate.c

```diff
@@ -2247,6 +2250,16 @@ static const annotate_entrydesc_t mailbox_builtin_entries[] =
         annotation_set_pop3showafter,
         NULL,
         NULL
+    },{
+        IMAP_ANNOT_NS "search-fuzzy-always",
+        ATTRIB_TYPE_STRING,
+        BACKEND_ONLY,
+        ATTRIB_VALUE_PRIV,
```

Comment:
When we talked about this on the conf call, I thought we concluded that this needed to use a shared value, not a private one?  But I don't understand/remember why (I don't think about annotations much), and maybe I'm misremembering?

Reply:
I don't remember that we concluded that, but I am fine with either. I just wouldn't see  a use case why one user would be able to toggle the setting for another user.

Reply:
Changed to shared annotation.

Reply:
I'm not sure either, but I guess it might be so that the admin process setting the flag from the web interface, and the user imapd process doing the search, both see the same annotation value, and not distinct ones private to each of them.